src/content-scripts/gemini-tracker/gemini-history-dom-observer.js

```
@@ -48,9 +186,128 @@
+
+        // Log details of significant mutations
+        let significantChanges = 0;
+        for (const mutation of mutations) {
+          if (mutation.type === "childList" && mutation.addedNodes.length > 0) {
+            for (const node of mutation.addedNodes) {
+              if (node.nodeType === Node.ELEMENT_NODE) {
+                // Only log if it seems like an important element
+                if (
+                  node.tagName.toLowerCase().includes("app") ||
+                  node.tagName.toLowerCase().includes("chat") ||
+                  (node.querySelector && node.querySelector("button, mat-icon, textarea"))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)